pkg/tagger/utils/expiry_test.go

```diff
@@ -0,0 +1,75 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package utils
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestUpdate(t *testing.T) {
+	expire := &Expire{
+		expiryDuration: 5 * time.Minute,
+		lastSeen:       make(map[string]time.Time),
+	}
+	testContainerID := "b2beae57bb2ada35e083c78029fe6a742848ff021d839107e2ede87a9ce7bf50"
```

Comment:
please remove references to containers, and that way you can use simpler test strings

Reply:
👌